### ISSUE TITLE
FLAG_SECURE doesn't work on some devices

### ISSUE THREAD

**Issue by ShamanisTe** [infil00p/cordova-crosswalk-engine#52]
I working on Cordova Application on Android.

The FLAG_SECURE don't allow user to take screenshot and when the application is in background, he have to see a white screen to protect information in the last view.

My problem is when I add **cordova-crosswalk-engine** in my project, the FLAG_SECURE working only on some devices... 

I try with :
- https://github.com/devgeeks/PrivacyScreenPlugin
- add _"getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);"_ before "**loadUrl**(..)" in my MainActivity.java

List devices tested with success :
Samsung Galaxy S5
Nexus 6 - 6.0.1
Motorola Moto 4G - 5.0.2

List devices tested with error : 
Samsung Galaxy Note 2 - 4.1.2
Sony Xperia Z5 - 6.0.1
Samsung Galaxy Ace 4 - 4.4.4

**Comment by fujunwei** [comment]
Do you mean it's work finely without Cordova Crosswalk plugin?

**Comment by ShamanisTe** [comment]
Yes, when i remove Cordova Crosswalk plugin, the FLAG_SECURE work correctly
